### ISSUE TITLE
File Manager Panel - Scroll Box

### ISSUE THREAD

**Issue by vslinuxdotnet** [4refr0nt/ESPlorer#57]
Hello, 

Can you please add in File Manager Panel, one scroll box with buttons (up and down) to be possible to open all files in list ?

Regards,

VS

**Comment by nedoskiv** [comment]
Yeah it will be very nice, I bet some java developper gonna do it in 6-7 lines of code :)

**Comment by avkplus** [comment]
Hi.
A scrollbar would be very useful when there are a lot of files. Can you implement a scrollbar?
Many thanks!
